docs/changes/uncertainty/11210.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+``Distribution`` instances can now be used as input to ``Quantity`` to
+initialize ``QuantityDistribution``.  Hence, ``distribution * unit``
```

Comment:
Should this instead be
```suggestion
initialize ``QuantityDistribution``.  Hence, ``distribution << unit``
```
?

Reply:
Both did not work before, in the sense that one did not get a `QuantityDistribution` out (but instead a `Quantity` with a structured dtype, which is less than useful:
```
In [7]: Distribution(np.ones((10,10)))
Out[7]: 
NdarrayDistribution([[1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.]]) with n_samples=10

In [8]: Distribution(np.ones((10,10))) * u.m
Out[8]: 
<Quantity [([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],),
           ([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],)] m>
```

With this PR:
```
In [3]: Distribution(np.ones((10,10))) * u.m
Out[3]: 
<QuantityDistribution [[1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.],
           [1., 1., 1., 1., 1., 1., 1., 1., 1., 1.]] m with n_samples=10>
```

Reply:
Now clarified.